docs/dashboards/overview-dashboard.asciidoc

```diff
@@ -19,7 +19,7 @@ image::images/live-feed-ov-page.png[Overview dashboard with live feed section hi
 
 Time-based histograms show the number of detections, alerts, and events that have occurred within the selected time range. To focus on a particular time, click and drag to select a time range, or choose a preset value. The *Stack by* menu lets you select which field is used to organize the data. For example, in the Alert trend histogram, stack by `kibana.alert.rule.name` to display alert counts by rule name within the specified time frame.
 
-TIP: Many {elastic-sec} histograms, graphs, and tables contain an *Inspect* button so you can examine the {es} queries used to retrieve data throughout the app.
+Hover over histograms, graphs, and tables to display an *Inspect* button (image:images/inspect-icon.png[Inspect icon,19,19]) or options menu (image:images/three-dot-icon.png[Three-dot menu icon,18,18]), allowing you to inspect the visualization's {es} queries, add it to a new or existing case, or open it in Lens for customization.
```

Comment:
```suggestion
Hover over histograms, graphs, and tables to display an *Inspect* button (image:images/inspect-icon.png[Inspect icon,19,19]) or options menu (image:images/three-dot-icon.png[Three-dot menu icon,18,18]). These allow you to inspect the visualization's {es} queries, add it to a new or existing case, or open it in Lens for customization.
```

Reply:
One more optional edit to make the second sentence actionable:

```suggestion
Hover over histograms, graphs, and tables to display an *Inspect* button (image:images/inspect-icon.png[Inspect icon,19,19]) or options menu (image:images/three-dot-icon.png[Three-dot menu icon,18,18]). Click these to inspect the visualization's {es} queries, add it to a new or existing case, or open it in Lens for customization.
```

Reply:
```suggestion
Hover over histograms, graphs, and tables to display an *Inspect* button (image:images/inspect-icon.png[Inspect icon,19,19]) or options menu (image:images/three-dot-icon.png[Three-dot menu icon,18,18]). Click to inspect the visualization's {es} queries, add it to a new or existing case, or open it in Lens for customization.
```